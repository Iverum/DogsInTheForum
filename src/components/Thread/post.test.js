import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Post from './post';

test('post is rendered with username and text', () => {
  const component = renderer.create(
    <Post username="Anonymous" text="Lorem ipsum" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Post username="Anonymous" text="Lorem ipsum" />, div);
});
