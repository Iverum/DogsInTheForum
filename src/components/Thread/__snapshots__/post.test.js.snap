exports[`test post is rendered with username and text 1`] = `
<div
  className="row">
  <div
    className="three columns">
    <h3>
      Anonymous
    </h3>
  </div>
  <div
    className="nine columns"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<p>Lorem ipsum</p>
      ",
      }
    } />
  <div
    className="dice-hand">
    <table
      className="u-full-width">
      <tbody />
    </table>
  </div>
</div>
`;
